src/lang/en/seat.php

```diff
@@ -23,6 +23,7 @@
 return [
 
     // General
+    'global'                => 'Global',
```

Comment:
maybe role would be more appropriate?

Reply:
Yeah global seems weird here. Could you perhaps explain what you intended with this?